reference/enterprise-api-ref/host.markdown

```diff
@@ -183,6 +185,34 @@ curl -k --user admin:admin -X GET https://hub.example.com/api/hosts/by-class
 }
 ```
 
+**inventoryFile=true**
+
```

Comment:
I think it could be worth noting that this returns the appropriate format for ansible inventory sourced from a file. The default response format (where this is false) returns in the format expected by an ansible inventory _script_.

@vpodzime ?

Reply:
I added a commit to add this, @vpodzime what do you think of the updated text?